pages/event/[eventSlug]/preScreeningMiddle/index.vue

```
@@ -0,0 +1,150 @@
+const route = useRoute();
+const eventStore = useEventStore();
+const router = useRouter();
+
+// Pega o slug do evento da URL
+const eventSlug = route.params.eventSlug as string;
+
+// Carrega as informações do evento
+const eventConfig = await eventStore.getEvent(eventSlug);
+
+// Você vai passar essas props de fora:
+const props = defineProps({
```

ESLint

[error] 'props' is assigned a value but never used.

(@typescript-eslint/no-unused-vars)